libcxx/test/std/input.output/iostream.format/output.streams/ostream.formatted/ostream.formatted.print/locale-specific_form.pass.cpp

```diff
@@ -86,10 +87,11 @@ test(std::stringstream& stream, std::string expected, test_format_string<char, A
   }
   // *** println ***
   {
-    expected += '\n'; // Tested last since it changes the expected value.
+    expected += "\n\n"; // Tested last since it changes the expected value.
     stream.str("");
     ;
     std::println(stream, fmt, std::forward<Args>(args)...);
+    std::println(stream);
     std::string out = stream.str();
     TEST_REQUIRE(out == expected,
                  TEST_WRITE_CONCATENATED(
```

Comment:
Please split in two tests, now if
` std::println(stream, fmt, std::forward<Args>(args)...); -> "\n\n"`
` std::println(stream)-> ""`
the test passes.

The same for the other tests, except the `println.sh.cpp`. That last test is a bit trickier so let's keep that as is.

Reply:
Or we can have println_blank_line.sh.cpp?

Reply:
I'm fine with an additional test too.